<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PyQt5 Example Application</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>194</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>90</y>
      <width>50</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextEdit1:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>134</y>
      <width>50</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextEdit2:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>125</y>
      <width>194</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>66</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber1">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>126</y>
      <width>124</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>9</number>
    </property>
    <property name="value" stdset="0">
     <double>3.141590000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider1">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>185</y>
      <width>188</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>18</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar1">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>238</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox1">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>220</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox2">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>240</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox3">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>260</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>75</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>124</y>
      <width>75</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>33</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>124</y>
      <width>33</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>492</x>
      <y>432</y>
      <width>156</width>
      <height>23</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox1">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>280</y>
      <width>187</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView1">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>310</y>
      <width>256</width>
      <height>135</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
